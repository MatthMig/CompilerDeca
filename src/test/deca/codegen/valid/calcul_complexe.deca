// Description :
//
//      Test d'un calcul complexe
// 
// Résultats : 
// 
//      N/A
//
// Historique :
// 
//      Créé le 07/06/2023 par gl03

{
    float y= (3*2/(4+6*(3+3))-3)*(2+1)-1+80;
    int x = ((3+6)%2)*(6-(3%4));
    println(y);
    println(x);
}