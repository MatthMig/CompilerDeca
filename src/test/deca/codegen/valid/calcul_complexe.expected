7.04500e+01
3
